robohive/envs/multi_task/common/kitchen/franka_kitchen.xml

```diff
@@ -47,14 +47,13 @@
             <geom type="box" group="3" pos='0 0 .142' size="0.02 0.10 0.03" contype="0" conaffinity="0" rgba=".9 .7 .95 1" euler="0 0 -.785"/>
         </body>
 
-        <site name='target' pos='0 0 0' size='0.1' rgba='0 2 0 .2'/>
+        <site name='target' type='box' size='.03 .07 .04' pos='-0.4 0.4 2.1' group='1' rgba='0 1 .4 0' euler="0 3.14 3.14"/>
```

Comment:
To make it a general solution 
1. we can keep the site in its default position. 
2. When teleOp is initialized, we should programmatically move this site to where the end effector is. That will avoid any initial jump in teleOp during connection to the robot.

Reply:
Done! It's working as expected, but I'm seeing some weird behaviour from the `rpFrankaRobotiqData-v0` environment:

The target site isn't placed *exactly* at the end effector. It's placed a little bit above + rotated compared to the actual location of the end effector.  For the Franka Kitchen environment, the goal site is centered exactly on the end effector.

Reply:
This seems like an issue with conventions. Double-check where all the site frames are (both position and orientations)

Reply:
It looks like we see this issue because the `end_effector` site is defined for the Franka gripper, and the Robotiq gripper doesn't redefine it. Because of the grippers' different dimensions, it doesn't align perfectly for the Robotiq gripper.  

- The overall difference is fairly small (see images below) and probably doesn't matter much for teleop.
- This issue can also be observed in the main branch and is not introduced in this PR.

![franka_gripper](https://github.com/vikashplus/robohive/assets/19668051/9217fae0-5867-4c6b-bc79-51089c0ee090)

![robotiq_gripper](https://github.com/vikashplus/robohive/assets/19668051/f5802dd6-3779-40d0-9e1e-b17dd54d360d)